src/main/groovy/pl/allegro/tech/build/axion/release/domain/scm/ScmInitializationOptions.groovy

```diff
@@ -12,23 +12,27 @@ class ScmInitializationOptions {
 
     final String remoteUrl
 
+    final boolean pushTagsOnly
+
     private static final String ATTACH_REMOTE_PROPERTY = 'release.attachRemote'
 
     private static final String FETCH_TAGS_PROPERTY = 'release.fetchTags'
 
-    ScmInitializationOptions(String remote, boolean fetchTags, boolean attachRemote, String remoteUrl) {
+    ScmInitializationOptions(String remote, boolean fetchTags, boolean attachRemote, String remoteUrl, boolean pushTagsOnly) {
```

Comment:
i wonder - this option might be set from cmd line as well (via `-Prelease.pusthTagsOnly`), so there would be no need to leak CI settings into buildscript